README.md

```diff
@@ -6,7 +6,11 @@ https://firecloud-ci.broadinstitute.org/
 
 ## Getting Started
 
+[ClojureScript](https://github.com/clojure/clojurescript) is used for the UI.
 We use the Leiningen build tool. To install, follow the installation instructions on the [Leiningen web site](http://leiningen.org/).
+The code incorporates usage of [react-cljs](https://github.com/dmohs/react-cljs) which is 
+a ClojureScript wrapper for React.  react-cljs largely follows React's top-level API 
```

Comment:
"react-cljs largely follows React's top-level API..." isn't entirely correct. More importantly, if the library were to change in some way that makes this untrue, it is somewhat unlikely that we would remember to update this sentence. Better to remove it and refer the reader to that page for the most up-to-date documentation.